.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,693 @@
+id: 

```

Comment:
@JohannesLampel it looks like this file doesn't include any `AddMilestone` actions - I thought that feature had already been added to PolicyBot?

Reply:
@wtgodbe this will be added next quarter, in the meantime can you merge this, we will have to decommission within 2weeks  the old FabricBot infra since there are s360 security issues that can't be resolved